nilearn/mass_univariate/permuted_least_squares.py

```diff
@@ -52,6 +53,8 @@ def normalize_matrix_on_axis(m, axis=0):
         ret = normalize_matrix_on_axis(m.T).T
     else:
         raise ValueError('axis(=%d) out of bounds' % axis)
+    if LooseVersion(np.__version__) > LooseVersion('1.13'):
```

Comment:
Actually, here the parameter legacy is implemented from 1.14. So, I should change this to fix appveyor.